index.html

```diff
@@ -5154,7 +5154,56 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
   </p>
 
 </section>
+<section id="verifiable-credential-media-type">
+  <h2>The <code>application/verifiable-credential+ld+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/verifiable-credential+ld+json</code> Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Credentials format.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>verifiable-credential+ld+json</td>
```

Comment:
```suggestion
      <td>vc+ld+json</td>
```

Also, see this:

https://github.com/w3c/vc-data-model/pull/1032#pullrequestreview-1325134836

Suggesting the following as a way to get this PR into the spec:

1. Rename `application/credential+ld+json` media type registration to `application/vc+ld+json`.
2. Register `application/vp+ld+json` as the media type for verifiable presentations. @brentzundel's argument is compelling, anything that contains a VC in it is a VP (whether or not the VP is secured or not, the mere existence of the VC in the VP makes it verifiable).
3. Use @mprorock's PR #1055 to define exactly what the expectations are around secured credentials and secured presentations, because at the end of the day, that's what's going to drive system behavior (not these unsecured things).
4. DO NOT register `application/credential+ld+json` and `application/presentation+ld+json` because they add little value.
5. Speak to the difference between credentials/presentations and verifiable credentials/verifiable presentations in the spec in a non-normative fashion.
6. Think about defining a `Credential` and `Presentation` RDF class after we work through 1-5 above.